dojo/templates/dojo/product.html

```diff
@@ -350,8 +350,16 @@ <h3 class="has-filters">
                         'Very Low': 1,
                         'None': 0
                       };
-                      return type === 'sort' ? criticals[getDojoExportValueFromTag(data, 'i', 'data-content')] :
-                            type === 'export' ? getDojoExportValueFromTag(data, 'i', 'data-content') : data;
+                      if (type === 'sort') {
+                        return criticals[getDojoExportValueFromTag(data, 'i', 'data-content')]
+                      }
+                      else if (type === 'export') {
+                          return getDojoExportValueFromTag(data, 'i', 'data-content')
+                      }
+                      else if (data != undefined||data != null) {
+                          return data
+                      }
+                      return ""
```

Comment:
Just a minor styling suggestion:

```suggestion
                      if (type === 'sort') {
                        return criticals[getDojoExportValueFromTag(data, 'i', 'data-content')]
                      } else if (type === 'export') {
                          return getDojoExportValueFromTag(data, 'i', 'data-content')
                      } else if (data != undefined||data != null) {
                          return data
                      }
                      return ""
```

Reply:
Agreed with this request

Reply:
Changes made in this [commit](https://github.com/DefectDojo/django-DefectDojo/pull/8765/commits/cc45195efc05563e006525dc5cdfe537ecf5f7dc)